### ISSUE TITLE
Our MAN (and get-help) should fall through to /usr/bin/man on failure to find something

### ISSUE THREAD

**Issue by jpsnover** [PowerShell/PowerShell#702]
We alias MAN to be get-help and that is fine BUT it has to work properly.  People need to be able to find their man pages.  

Our behavior is to find the help associated with a name and if we can't, we'll do a search.  I think we need to look for the native help between those two steps.

**Comment by joeyaiello** [comment]
FYI, `man` is no longer a problem here since we decided to remove the *nix-y aliases on Linux/macOS. Still, we should do something about the Get-Help scenario.

**Comment by andschwa** [comment]
Enhancing the `Get-Help` cmdlet to fall-through to *nix man pages would be a great Hacktoberfest pull request!

**Comment by rkeithhill** [comment]
The problem now is that you've trained Windows PowerShell users to use `man` for viewing help. When they get to PS Core on Linux, they'll run into this:
```
PS /home/hillr> man about_requires
No manual entry for about_requires
```
:-(

**Comment by davidBar-On** [comment]
A note to whoever will try to fix this issue - see PR #15343 with a proposed fix for the issue (was automatically closed).
